src/gui/mainwindow.cpp

```diff
@@ -1713,6 +1716,7 @@ void MainWindow::createTrayIcon(const int retries)
         connect(m_systrayIcon, &QSystemTrayIcon::messageClicked, this, &MainWindow::balloonClicked);
 
         m_systrayIcon->show();
+        emit systemTrayIconCreated();
```

Comment:
Why not just put `m_ui->actionLock->setVisible(true)`?

Reply:
IMO this function is for tray icon. I feel like `actionLock` doesn't really belong in it.